tests/Constraints/CoerciveTest.php

```diff
@@ -128,7 +128,7 @@ public function getValidCoerceTests()
                   "boolean":"true",
                   "object":{},
                   "array":[],
-                  "null":null,
+                  "null":"",
```

Comment:
I don't understand. Can you provide an example? 

I believe I originally created this file by duplicating BasicTypesTest.